src/Components/AddMovieBtn/AddMovieBtn.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { AddMovieBtn } from './AddMovieBtn';
+
+describe('AddMovieBtn', () => {
+  it('should match AddMovieBtn snapshot', () => {
+    const { asFragment } = render(<AddMovieBtn handleClick={() => {}} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should trigger handleClick on button click', () => {
+    const handleClick = jest.fn();
+
+    render(<AddMovieBtn handleClick={handleClick} />);
+    userEvent.click(screen.getByRole('button'));
```

Comment:
It is good practice to cover events to act faction from react library. Especially if it changes state.